nifti2/nifti_tool.c

```diff
@@ -6710,7 +6721,8 @@ int act_disp_ci( nt_opts * opts )
          fprintf(stdout,")\n");
       }
 
-      disp_raw_data(data, nim->datatype, len / nim->nbyper, space, 1);
+      /* should we change disp_raw_data to allow for 64-bit nvalues? */
```

Comment:
We should, and I did in https://github.com/NIFTI-Imaging/nifti_clib/pull/165

Reply:
Oh right, though the counter was only of type int.
Still, it would be more clear to put functionality enhancement PRs separate from code cleanup ones.  This looked more like "well, we've altered the type, so now it should propagate".  It would be fine to go 64-bit here, but if you want that, maybe just make it a separate change.

Reply:
>Still, it would be more clear to put functionality enhancement PRs separate from code cleanup ones.

I did split my `-Wshorten-64-to-32` fixes into 2 PRs:

- #157 which changed no APIs, that this PR #171 is partly reverting
- #165 which does change APIs like `disp_raw_data` above.  It's awaiting review.

Reply:
If you would like to make the 64-bit update for this, please do not change library APIs in that PR.  Modifying types in nifti_tool is okay, modifying them in the library is a bigger deal.  Let's keep them separate.  Any library updates should be done in the context of heading to a 4.x.x release.  We should make useful updates in the current 3.x.x version, first.